contracts/src/periphery/cert/router/EigenDACertVerifierRouter.sol

```diff
@@ -0,0 +1,79 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.9;
+
+import {IEigenDACertVerifier} from "src/periphery/cert/interfaces/IEigenDACertVerifier.sol";
+import {IEigenDACertVerifierRouter} from "src/periphery/cert/interfaces/IEigenDACertVerifierRouter.sol";
+import {OwnableUpgradeable} from "lib/openzeppelin-contracts-upgradeable/contracts/access/OwnableUpgradeable.sol";
+
+contract EigenDACertVerifierRouter is IEigenDACertVerifierRouter, OwnableUpgradeable {
+    mapping(uint32 => address) public certVerifiers;
+    uint32[] public certVerifierABNs;
+
+    event CertVerifierAdded(uint32 indexed abn, address indexed certVerifier);
+
+    error ABNNotInFuture(uint32 abn);
+    error ABNNotGreaterThanLast(uint32 abn);
+    error InvalidCertLength();
+    error NoCertVerifierAvailable();
+    error NoCertVerifierFound(uint32 rbn);
+
+    /// IEigenDACertVerifierRouter ///
+
+    function checkDACert(bytes calldata certBytes) external view returns (uint8) {
+        return IEigenDACertVerifier(getCertVerifierAt(_getRBN(certBytes))).checkDACert(certBytes);
+    }
+
+    function getCertVerifierAt(uint32 abn) public view returns (address) {
+        return certVerifiers[_findClosestRegisteredABN(abn)];
```

Comment:
nit: 
```suggestion
        return certVerifiers[_findClosestRegisteredABNIndex(abn)];
```

Reply:
Don't think this is correct. The valuer returned is not an "index", but is the actual ABN that is closest to the input ABN.